src/main/java/teammates/storage/api/FeedbackQuestionsDb.java

```diff
@@ -156,7 +160,7 @@ public FeedbackQuestionAttributes getFeedbackQuestion(
      * * {@code newAttributes.getId()} is non-null and
      *  correspond to an existing feedback question. <br>
      */
-    public void updateFeedbackQuestion(FeedbackQuestionAttributes newAttributes) throws InvalidParametersException, EntityDoesNotExistException {
+    public void updateFeedbackQuestion(FeedbackQuestionAttributes newAttributes, boolean keepUpdateTimestamp) throws InvalidParametersException, EntityDoesNotExistException {
```

Comment:
Updating the header comment here and adding some short comment in the other `updateFeedbackQuestion` method to denote that it's a variant would be good.